engine/src/main/scala/cromwell/webservice/routes/wes/WesState.scala

```diff
@@ -0,0 +1,54 @@
+package cromwell.webservice.routes.wes
+
+import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport
+import cromwell.core._
+import spray.json.{DefaultJsonProtocol, JsString, JsValue, RootJsonFormat}
+
+object WesState {
+  sealed trait WesState extends Product with Serializable
+  case object UNKNOWN extends WesState
+  case object QUEUED extends WesState
+  case object INITIALIZING extends WesState
+  case object RUNNING extends WesState
+  case object PAUSED extends WesState
+  case object COMPLETE extends WesState
+  case object EXECUTOR_ERROR extends WesState
+  case object SYSTEM_ERROR extends WesState
+  case object CANCELED extends WesState
+  case object CANCELING extends WesState // Before any shorebirds worry about the spelling, it really is CANCELING
```

Comment:
Shorebird [approved](https://books.google.com/ngrams/graph?content=canceling%2Ccancelling&year_start=1800&year_end=2018&corpus=5&smoothing=3&share=&direct_url=t1%3B%2Ccanceling%3B%2Cc0%3B.t1%3B%2Ccancelling%3B%2Cc0).

Reply:
You wouldn't believe how much combined energy was spent debating one or two `l`s in this

Reply:
Coming from a family who's motto is "why have just one L when you can have two", I heartilly disagree with this choice :)

Reply:
Unfortunately that choice is not ours to make. If only you participated more heavily in GA4GH :)